### ISSUE TITLE
Corectrl doesn't work with kernel 5.15.5

### ISSUE THREAD

**Issue by RodriMora** [pop-os/pop#2034]
**Distribution (run `cat /etc/os-release`):**
```
 NAME="Pop!_OS"
VERSION="21.04"
ID=ubuntu
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 21.04"
VERSION_ID="20.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=hirsute
UBUNTU_CODENAME=hirsute
LOGO=distributor-logo-pop-os
```

**uname -r**
`5.15.5-76051505-generic`

note: I had the version ID edited to be able to install some AMD drivers
 
 

**Issue/Bug Description:**
The advanced options of Corectrl software stopped working on this version of the kernel. It's also reported here https://gitlab.com/corectrl/corectrl/-/issues/230
It seems to be fixed on kernel 5.15.6 and 5.16

**Steps to reproduce (if you know):**
Have an AMD card (rx 580 in my case)
Install corectrl with sudo apt install corectrl
Add sudo kernelstub -a "amdgpu.ppfeaturemask=0xffffffff" to have advanced features of corectrl
Open program and see that there are no advanced options

**Expected behavior:**
To have advanced core/voltage options

**Comment by jacobgkau** [comment]
Thank you for the report. If this is fixed by kernel 5.15.6, then it should be fixed once https://github.com/pop-os/linux/pull/85 is merged.

**Comment by RodriMora** [comment]
I close this, using oldkern as a workaround and waiting for 5.15.6 to be merged
